src/app/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from "react";
+import Routing from "pages/routes/ui/routes";
+import { withProviders } from "./providers";
+import { onAppLoaded } from "entities/session";
+import "./index.css";
+
+const App = () => {
+  useEffect(() => {
```

Comment:
if yoy want use loadin - try use pending in fx.  forexample const loading = useStore(model.getCompanyDetailsFx.pending);

Reply:
In this exact example function is needed for authentication. but thanks, I was questioning myself right seconds before your review anout loading state! Thank you so much!